solr/core/src/test/org/apache/solr/update/processor/SignatureUpdateProcessorFactoryTest.java

```diff
@@ -179,20 +179,20 @@ public void run() {
       threads2[i].setName("testThread2-" + i);
     }
 
-    for (int i = 0; i < threads.length; i++) {
-      threads[i].start();
+    for (Thread element : threads) {
+      element.start();
     }
 
-    for (int i = 0; i < threads2.length; i++) {
-      threads2[i].start();
+    for (Thread item : threads2) {
+      item.start();
     }
 
-    for (int i = 0; i < threads.length; i++) {
-      threads[i].join();
+    for (Thread value : threads) {
+      value.join();
     }
```

Comment:
Curious about the `element/item/value` naming choices here, would `thread` be clearer?